QuizBowlDiscordScoreTracker/TeamManager/ITeamManager.cs

```diff
@@ -12,5 +12,7 @@ public interface ITeamManager
         Task<string> GetTeamIdOrNull(ulong userId);
 
         Task<IReadOnlyDictionary<string, string>> GetTeamIdToNames();
+
+        public void ReloadTeamRoles(out string message);
```

Comment:
Remove this method, since we only need it in IByRoleTeamManager. We make sure that the team manager is this type in the command handler, so it's safe to remove. By removing this, we can also remove the methods from SoloOnlyTeamManager and ByCommandTeamManager.